app/editor/src/features/admin/notifications/constants/defaultNotification.ts

```diff
@@ -4,15 +4,17 @@ export const defaultNotification: INotificationModel = {
   id: 0,
   name: '',
   description: '',
-  ownerId: 0,
-  template: '',
-  filter: {},
-  settings: {},
+  templateId: 0,
+  settings: {
+    searchUnpublished: false,
+    size: 0,
+  },
+  query: {},
   isEnabled: false,
   isPublic: false,
   sortOrder: 0,
   subscribers: [],
   notificationType: NotificationTypeName.Email,
-  requireAlert: false,
+  alertOnIndex: false,
```

Comment:
This replaces the logic of `requireAlert`.  Now if a notification requires an alert it simply includes that as part of its filter.